### DISCUSSION TITLE
vercel platform

### DISCUSSION THREAD

**Discussion by kazewaze** [vercel/next.js#41935] [Help] [answered]
After my personal access token expired on github and I created a new one, the next.js project hosted on vercel is no longer deploying the latest changes. I'm assuming it's because the token changed but I am not sure how to fix this issue.

**Comment by icyJoseph** [comment, marked as answer]
I guess you gotta re-authorize Vercel? Log out and log back in to Vercel with your GH account. Assuming of course that that's the problem. 

It might also be that you are using the token as an ENV var in your project, and need to update it in the Dashboard > Env Vars interface, and re-deploy your site.

**Reply by kazewaze** [reply]
Thanks man, I always stayed logged in to vercel but never thought to logout and log back in. That worked!
